Scripts/ManualSubmissionScript/manual-submission.py

```diff
@@ -0,0 +1,77 @@
+import icat
+from settings import ACTIVE_MQ
+from settings import ICAT
```

Comment:
@jsws The settings.py.template should be updated to use `ICAT = {...}` instead of the current `ICAT_SETTINGS = {...}`

Reply:
Fixed in c4c1561